tests/test_order.py

```diff
@@ -95,12 +107,11 @@ def test_should_get_assets_if_fulfilled(self, auth_mock, requests_mock: req_mock
                     "id": constants.ASSET_ORDER_ID,
                 }
             ],
-            "totalPages": 1,
-            "totalElements": 1,
+            "totalPages": 2,
         }
         requests_mock.get(url=url_asset_info, json=asset_info)
-        order_obj = order.Order(auth=auth_mock, order_id=constants.ORDER_ID)
-        (asset_obj,) = order_obj.get_assets()
+        order_obj = order.Order(order_id=constants.ORDER_ID)
+        asset_obj = order_obj.get_assets()[0]
```

Comment:
Let's keep the previous approach here, it verifies also that there's only one item there

Reply:
That's exactly what we're trying to test. If for some reason it would return more than expected this test must fail